### DISCUSSION TITLE
I found Some gp file overlap when playing horizontally，The cause is unknown

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#1089] [Q&A] [answered]
like this img
![未命名-1](https://user-images.githubusercontent.com/17884664/210233245-a9f5fd70-4734-4ed2-aca7-555a3a55cea3.jpg)

the gp file
[61d9c385-ab20-4587-9a28-ef90ad3b8468.zip](https://github.com/CoderLine/alphaTab/files/10331666/61d9c385-ab20-4587-9a28-ef90ad3b8468.zip)

**Comment by Danielku15** [comment, marked as answer]
Seems like a bug. I added https://github.com/CoderLine/alphaTab/issues/1090
